liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -55,6 +55,13 @@ std::string SourceReferenceFormatter::formatErrorInformation(Error const& _error
 	);
 }
 
+enum class Severity//have to define if used in switch
```

Comment:
You should use the existing `Severity` enum defined in `liblangutil/Exceptions.h`. If you define a new enum here, the compiler won't consider the two the same and you'll get an error.

Reply:
Fixed 👍